src/ui/public/partials/saved_object_finder.html

```diff
@@ -78,6 +78,7 @@
           ng-blur="finder.hitBlur($event)"
           ng-click="finder.preventClick($event)">
           <span aria-hidden="true" class="finder-type fa" ng-if="hit.icon" ng-class="hit.icon"></span>
+          <div class="kuiIcon fa-flask ng-scope" ng-if="hit.type && hit.type.isExperimental"></div>
```

Comment:
`ng-if="hit.type.isExperimental"` is enough. Angular will automatically safe traverse that object, and not fail if `hit.type` isn't defined.

Reply:
seriously? that's awesome! :)